app/helpers/application_helper.rb

```diff
@@ -23,7 +23,12 @@ def avatar(email = nil)
     else
       gravatar_id = Digest::MD5.hexdigest(email.downcase)
       gravatar_size = Rails.application.config.gravatar_size
-      tag :image, src: "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(root_url + default)}"
+      if Rails.env.development?
+        gravatar_default = "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(default)}"
+      else
+        gravatar_default = "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(root_url + default)}"
+      end
```

Comment:
The point of keeping things DRY, is to not repeat stuff unless you have to. We don't need to repeat `http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(` here.